datavirt64/hibernate-on-top-of-teiid/README.md

```diff
@@ -0,0 +1,119 @@
+
+## System requirements
+
+If you have not done so, please review the System Requirements [../README.md](../README.md)
+
+This example produces a WAR that is designed to be run on JBoss Enterprise Application Platform 6 or JBoss AS 7.
+
+> NOTE: Correct Dependencies - Please note this example does not support working with Hibernate 3.  
+
+## Build the project
+
+- Before building, need to set the username/password in the hibernate config file
+
+The src/main/webapp/WEB-INF/hibernate4-teiid-quickstart-ds.xml file needs to have the user-name and password changed from user/user to the
+teiid user you wish to connect with.
+
+- Open a command line and navigate to the root directory of this quickstart
+
+	*   mvn -s ../settings.xml clean install
+
+## Setup
+
+1) Run the setup in dynamicvdb-datafederation quick start
+
+2) Make sure to start the server, if not already
+
+	To start the server, open a command line and navigate to the "bin" directory under the root directory of the JBoss server and run:
+	
+	For Linux:   ./standalone.sh	
+	for Windows: standalone.bat
+
+	To use one of the high-available (ha) configurations for clustering, append the following arguments to the command to specify the configuration
+		
+	-c {configuration.file} 
+
+[source,xml]
+.*Example*
+----
+./standalone.sh -c standalone-ha.xml
+----
+
+	
+3) VDB Deployment:
+
+-  run the following CLI script
+
+	-	cd to the $JBOSS_HOME/bin directory
+	-	execute:  ./jboss-cli.sh --connect --file=../quickstarts/hibernate-on-top-of-teiid/src/scripts/deploy_vdb.cli 
+
+
+4) Deploy the web application
+
+-  run the following CLI script
+
+	-	cd to the $JBOSS_HOME/bin directory
+	-	execute:  ./jboss-cli.sh --connect --file=../quickstarts/hibernate-on-top-of-teiid/src/scripts/deploy_war.cli 
+
+
+5)  Open the admin console to make sure the VDB is deployed
+
+	*  open a brower to http://localhost:9990/console
+
+6)  See "Query Demonstrations" below to demonstrate data federation.
+
+
+## Undeploy artifacts
+
+1)  To undeploy the web application run the following command:
+
+	-	cd to the $JBOSS_HOME/bin directory
+	-	execute:  ./jboss-cli.sh --connect --file=../quickstarts/hibernate-on-top-of-teiid/src/scripts/remove_war.cli 
+
+	
+2)  To undeploy the Teiid VDB, run the following command:
+
+	-	cd to the $JBOSS_HOME/bin directory
+	-	execute:  ./jboss-cli.sh --connect --file=../quickstarts/hibernate-on-top-of-teiid/src/scripts/remove_vdb.cli 
+
+	
+## Query Demonstrations
+
+### Access the application 
+
+The application will be running at the following URL: http://localhost:8080/hibernate-on-top-of-teiid/.
+
+The page should display a list of products.
+
+To add (Register) a new product, enter the following:
+
+-  Product ID  (must be unique)
+-  Company Name
+-  Stock Symbol
+
+> NOTE: the Stock Symbol entered must exist in the marketdata-price.txt file.  The file can be found at ../dynamicvdb-datafederation/src/teiidfiles/data/marketdata-price.txt
```

Comment:
/dynamicvdb-datafederation/**src**/teiidfiles/data/marketdata-price.txt no longer exist. There should be /dynamicvdb-datafederation/**app/data**/teiidfiles/data/marketdata-price.txt